ui-v2/src/components/concurrency/global-concurrency-view/data-table/actions-cell.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button } from "@/components/ui/button";
+import {
+	DropdownMenu,
+	DropdownMenuContent,
+	DropdownMenuItem,
+	DropdownMenuLabel,
+	DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+import { Icon } from "@/components/ui/icons";
+import { type GlobalConcurrencyLimit } from "@/hooks/global-concurrency-limits";
+import { useToast } from "@/hooks/use-toast";
+import { CellContext } from "@tanstack/react-table";
+
+type Props = CellContext<GlobalConcurrencyLimit, unknown> & {
+	onEditRow: (row: GlobalConcurrencyLimit) => void;
+	onDeleteRow: (row: GlobalConcurrencyLimit) => void;
+	onResetRow: (row: GlobalConcurrencyLimit) => void;
+};
+
+export const ActionsCell = ({
+	onEditRow,
+	onDeleteRow,
+	onResetRow,
+	...props
+}: Props) => {
+	const { toast } = useToast();
+
+	const handleCopyId = (id: string | undefined) => {
+		if (!id) {
+			throw new Error("'id' field expected in GlobalConcurrencyLimit");
+		}
+		void navigator.clipboard.writeText(id);
+		toast({ title: "Name copied" });
+	};
+
+	const row = props.row.original;
```

Comment:
For some reason in the schema `id` is optional for a GET response.
Even though it _should_ be expected for retrieving data?

Reply:
That is odd. We'll have to look at our typing on the Python side to see if we can fix that.